src/pages/login/index.tsx

```diff
@@ -45,8 +45,8 @@ const Login: FC = () => {
   useEffect(() => {
     const authenticate = async () => {
       try {
-        if (isValidSession(session)) {
-          await router.push(pages.home);
+        if (isUnverifiedSession(session)) {
```

Comment:
what is an "unverified session"? is it when the wallet is connected but a username hasn't been set? if so consider renaming the function to something like `awaitingUsername(session)` or `missingUsername(session)` to make it clear